chart/README.md

```diff
@@ -0,0 +1,1910 @@
+| ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- | -------------------------------------- |
+| `workspaceSubscriber.image.registry`                        | workspace-subscriber image registry                                                                           | `gcr.io`                               |
+| `workspaceSubscriber.image.repository`                      | workspace-subscriber image repository                                                                         | `carto-onprem-artifacts/workspace-api` |
+| `workspaceSubscriber.image.tag`                             | workspace-subscriber image tag (immutable tags are recommended)                                               | `2021.12.20`                           |
+| `workspaceSubscriber.image.pullPolicy`                      | workspace-subscriber image pull policy                                                                        | `IfNotPresent`                         |
+| `workspaceSubscriber.image.pullSecrets`                     | workspace-subscriber image pull secrets                                                                       | `[]`                                   |
+| `workspaceSubscriber.replicaCount`                          | Number of workspace-subscriber replicas to deploy                                                             | `1`                                    |
+| `workspaceSubscriber.containerPorts.http`                   | workspace-subscriber HTTP container port                                                                      | `3000`                                 |
+| `workspaceSubscriber.livenessProbe.enabled`                 | Enable livenessProbe on workspace-subscriber containers                                                       | `true`                                 |
+| `workspaceSubscriber.livenessProbe.initialDelaySeconds`     | Initial delay seconds for livenessProbe                                                                       | `10`                                   |
+| `workspaceSubscriber.livenessProbe.periodSeconds`           | Period seconds for livenessProbe                                                                              | `30`                                   |
+| `workspaceSubscriber.livenessProbe.timeoutSeconds`          | Timeout seconds for livenessProbe                                                                             | `5`                                    |
+| `workspaceSubscriber.livenessProbe.failureThreshold`        | Failure threshold for livenessProbe                                                                           | `5`       
```

Comment:
```suggestion
In case you want to add extra environment variables (useful for advanced operations like custom init scripts), you can use the `*.extraEnvVars` property. For instance, to add extra environment variables on lds-api containers: